serverless/src/index.ts

```diff
@@ -161,6 +175,25 @@ function lambdaHasDynamicallyGeneratedName(lambdas: LambdaFunction[]) {
   return dynmicallyNamedLambdas;
 }
 
+/**
+ * Builds the CloudFormation Outputs containing the alphanumeric key, description,
+ * and value (URL) to the function in Datadog
+ */
+function addOutputLinks(outputs: Outputs, lambdas: LambdaFunction[], site: string) {
+  for (const lambda of lambdas) {
+    const functionKey = lambda.key;
+    const key = `DatadogMonitor${functionKey}`.replace(/[^a-z0-9]/gi, "");
```

Comment:
Are we able to include hyphens here? If so, could we do something like

`DatadogServerlessView-${functionKey}`

Reply:
Unfortunately no, only alphanumeric characters are allowed. Can change it to `DatadogServerlessView${functionKey}` if that is preferred though.